CONTRIBUTING.md

```diff
@@ -170,14 +170,11 @@ Apart from that, we generally recommend adhering to the following guideline:
   ```
 
 - Unless another more specific name is suitable, use our default naming convention for 
-  `attrs` built-ins (defaults, converters, validators):
+  `attrs` defaults and validators:
   ```python
   @my_attribute.default
   def _default_my_attribute(self): ...
   
-  @my_attribute.converter
-  def _convert_my_attribute(self): ...
```

Comment:
what happened that you removed this?

Also, considering we said lets add a small sentence encouraging contributors to rebase, should we add it right away?

Reply:
Because it doesn't exist 😄 There is no such thing as a decorator version for converter. This is one of things on the attrs roadmap though, together with an overhaul of the converters such that they support the same three-argument version as validators do (also, there is a debate of improving the converter-validator capabilities altogether).

Regarding rebasing: have just push a draft. Can you have a look and let me know if OK?